index.bs

```diff
@@ -400,6 +478,49 @@ with parameters |session| and |capabilities| is:
 
 </div>
 
+Sessions {#sessions}
+==========================
+
+## status ## {#command-status}
```

Comment:
I added an initial attempt to group commands, using the term "module" as suggested. I agree this might be a better choice of name to help differentiate from CDP. We can refine this concept as we go but I think what's here is good enough for command dispatch.